modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -32,6 +32,8 @@ import (
 	"sync"
 	"time"
 
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+
 	"go.uber.org/zap"
 	"golang.org/x/net/http/httpguts"
```

Comment:
If my memory serves me right, the linter prefers them sorted by length. Move the `otelhttp` to below `httpguts` without any spaces in between.

Reply:
that's a good hint, thank you - I was frantically installing random linters to make golangci-lint work locally :D 

btw: is there a guide on how to automatically format it into shape and what my environment needs to make that happen?

Reply:
If you run `golangci-lint run --fix`, it should take care of most of them. The error in CI console also gives the command to run to fix it. I believe the Go plugin of vscode manages the formatting as well.